helm/azure-operator-chart/templates/deployment.yaml

```diff
@@ -13,17 +13,16 @@ spec:
   selector:
     matchLabels:
       app: azure-operator
+      version: {{ .Chart.Version }}
```

Comment:
We didn't have this before, so it makes me wonder if it needs to be changed. Before deploying the flattened version, we first would need to re-deploy the `thiccc` branch on all installations so it starts using both labels right?